internal/pkg/cli/svc_init.go

```diff
@@ -303,21 +281,42 @@ func (o *initSvcOpts) askDockerfile() (isDfSelected bool, err error) {
 	return true, nil
 }
 
-func (o *initSvcOpts) askSvcPort(parsedPorts []uint16) error {
+func (o *initSvcOpts) askSvcPort() (err error) {
+	// See if we can get a healthcheck from the dockerfile.
+	o.setupParser(o)
+	// Check for a valid healthcheck and add it to the opts.
+	var hc *manifest.ContainerHealthCheck
+	hc, err = o.parseHealthCheck()
```

Comment:
I think it will fail if it is using image (which is not what we expected because we don't need to parse healthcheck if it is using image).